crates/zoon/src/element/button.rs

```diff
@@ -8,6 +8,60 @@ use std::marker::PhantomData;
 
 make_flags!(Label, OnPress);
 
+/// Html Button for your web page.
```

Comment:
I'm not sure how to clearly distinguish Zoon Button and HTML `<button>` in documentation.
The problem: _"/// Html Button for your web page."_ may imply that Button is `<button>` or a wrapper for `<button>` but it's actually `<div>`.

Reply:
Thank you for the explanations and the feedback :wink: 

I will make the update :smile:

Reply:
I updated with better explanations.